newIDE/electron-app/app/DebuggerServer.js

```diff
@@ -62,4 +62,4 @@ module.exports = {
       cb(err);
     });
   },
-};
+};
```

Comment:
It looks like you have taken an extra newLine out of the file. Can you revert that? Having the extra line at the end of each js file is a weird javascript thing that is nice to have. Maybe this will help explain why
https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

In any case, you might notice we are doing it in Gdevelop. Prettier should be doing it automatically for you

Reply:
@blurymind sure I thought prettier in my IDE does weird things but I am extremely sorry about this, I will fix it right away